cloud/amazon/s3.py

```diff
@@ -409,6 +412,10 @@ def main():
     s3_url = module.params.get('s3_url')
     src = module.params.get('src')
 
+    for acl in module.params.get('permission').split(','):
```

Comment:
You mean make permission accept a list as well and convert string to list? I was thinking backwards compatibility.